keps/sig-scheduling/1451-multi-scheduling-profiles/README.md

```diff
@@ -305,12 +293,160 @@ Note that we don't require a feature gate as users already have to opt-in by usi
 Scheduling profiles will graduate to beta altogether with the graduation of
 the `kubescheduler.config.k8s.io` configuration API to Beta.
 
-See [KEP 785](/keps/sig-scheduling/785-scheduler-component-config-api) for
-more information.
+See [KEP 785] for more information.
+
+Additionally, we will include the scheduler profile as a field in the
+metrics related to scheduling attempts (counts and latency).
+
+[KEP 785]:(https://git.k8s.io/enhancements/keps/sig-scheduling/785-scheduler-component-config-api)
+
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+
+  - [ ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name:
+    - Components depending on the feature gate:
+  - [x] Other
+    - Describe the mechanism:
+    
+      Modify kube-scheduler configuration file to use a single profile.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+      
+      Yes, but only kube-scheduler.
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled)
+      
+      No
+
+* **Does enabling the feature change any default behavior?**
+
+  No, as long as a profile with the name of `default-scheduler` is kept.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  
+  Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  N/A.
+
+* **Are there any tests for feature enablement/disablement?**
+
+  There are test (unit and integration) exercising single a multiple profiles.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+
+  The scheduler errors and exits during start up. Existing workloads are not
+  affected.
+
+* **What specific metrics should inform a rollback?**
+
+  Metric "schedule_attempts_total" remaining at zero when new pods are added.
+
+* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+
+  N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features,
+  APIs, fields of API types, flags, etc.?**
+
+  See [KEP 785].
```

Comment:
I wouldn't mix those two things, those are two separate proposals (we can do this one without doing the other one in theory).

Reply:
Graduating this feature to beta goes along with graduating its API to beta.

On the other hand, API deprecation is independent.